### PR TITLE
python311Packages.{jax,jaxlib,jaxlib-bin}: 0.4.19 -> 0.4.20

### PR DESCRIPTION
**Pull request by GaetanLepage**
## Description of changes

Update `jax` packages.

cc @samuela 

## Things done

- Built on platform(s)
  - [x] x86_64-linux
  - [x] aarch64-linux
  - [ ] x86_64-darwin
  - [ ] aarch64-darwin
- For non-Linux: Is sandboxing enabled in `nix.conf`? (See [Nix manual](https://nixos.org/manual/nix/stable/command-ref/conf-file.html))
  - [ ] `sandbox = relaxed`
  - [ ] `sandbox = true`
- [ ] Tested, as applicable:
  - [NixOS test(s)](https://nixos.org/manual/nixos/unstable/index.html#sec-nixos-tests) (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
  - and/or [package tests](https://nixos.org/manual/nixpkgs/unstable/#sec-package-tests)
  - or, for functions and "core" functionality, tests in [lib/tests](https://github.com/NixOS/nixpkgs/blob/master/lib/tests) or [pkgs/test](https://github.com/NixOS/nixpkgs/blob/master/pkgs/test)
  - made sure NixOS tests are [linked](https://nixos.org/manual/nixpkgs/unstable/#ssec-nixos-tests-linking) to the relevant packages
- [ ] Tested compilation of all packages that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review rev HEAD"`. Note: all changes have to be committed, also see [nixpkgs-review usage](https://github.com/Mic92/nixpkgs-review#usage)
- [ ] Tested basic functionality of all binary files (usually in `./result/bin/`)
- [23.11 Release Notes](https://github.com/NixOS/nixpkgs/blob/master/nixos/doc/manual/release-notes/rl-2311.section.md) (or backporting [23.05 Release notes](https://github.com/NixOS/nixpkgs/blob/master/nixos/doc/manual/release-notes/rl-2305.section.md))
  - [ ] (Package updates) Added a release notes entry if the change is major or breaking
  - [ ] (Module updates) Added a release notes entry if the change is significant
  - [ ] (Module addition) Added a release notes entry if adding a new NixOS module
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md).

### CODE REVIEW

**Comment by GaetanLepage** [comment]
Currently, `jaxlib` fails to build (`boringssl` and `xla` issues)

**Comment by GaetanLepage** [comment]
Actually, I am not against a bit of help for building `jaxlib` @samuela :/
The issue seems to come from changes in XLA (the problematic `jax` commit is a bump in the XLA version).
Also, you can notice that I have needed to remove `boringssl` from the `tf_system_libs` as it was giving me an error.

**Comment by BillHuang2001** [comment]
I run into the same error while building the latest jax with rocm.
I can confirm that the problem was introduced in https://github.com/google/jax/commit/48ef80bc396cd12c3cde5273c0120d7b0fe21496, which updated it's xla version, so it's an XLA problem. But I'm not an expert in bazel, so I don't know what xla update causes this problem.

**Comment by samuela** [comment]
@GaetanLepage @BillHuang2001 are these issues that we could reasonably report upstream or are they caused by peculiarities with the nix build environment?

**Comment by BillHuang2001** [comment]
Delete this following patch seems to solve the problem, `jaxlib` will build, `jax` will also build, but failed in one unit test `tests/qdwh_test.py:testQdwhWithTinyElement`, but I don't think that's our problem.
```
      '' + (if stdenv.cc.isGNU then ''
        sed -i 's@-lprotobuf@-l:libprotobuf.a@' ../output/external/xla/third_party/systemlibs/protobuf.BUILD
        sed -i 's@-lprotoc@-l:libprotoc.a@' ../output/external/xla/third_party/systemlibs/protobuf.BUILD
      '' else if stdenv.cc.isClang then ''
        sed -i 's@-lprotobuf@${pkgs.protobuf}/lib/libprotobuf.a@' ../output/external/xla/third_party/systemlibs/protobuf.BUILD
        sed -i 's@-lprotoc@${pkgs.protobuf}/lib/libprotoc.a@' ../output/external/xla/third_party/systemlibs/protobuf.BUILD
      '' else throw "Unsupported stdenv.cc: ${stdenv.cc}");
```
@GaetanLepage is this patch still needed?

**Comment by GaetanLepage** [comment]
> Delete this following patch seems to solve the problem, `jaxlib` will build, `jax` will also build, but failed in one unit test `tests/qdwh_test.py:testQdwhWithTinyElement`, but I don't think that's our problem.

This patch was indeed what was preventing the build to work.

I removed it and it seems to work fine.
Actually, I haven't observed a single test failure.

**Comment by GaetanLepage** [comment]
Result of `nixpkgs-review pr 265417` run on x86_64-linux [1](https://github.com/Mic92/nixpkgs-review)
<details>
  <summary>28 packages marked as broken and skipped:</summary>
  <ul>
    <li>python310Packages.distrax</li>
    <li>python310Packages.distrax.dist</li>
    <li>python310Packages.dm-sonnet</li>
    <li>python310Packages.dm-sonnet.dist</li>
    <li>python310Packages.elegy</li>
    <li>python310Packages.elegy.dist</li>
    <li>python310Packages.rlax</li>
    <li>python310Packages.rlax.dist</li>
    <li>python310Packages.tensorflow-datasets</li>
    <li>python310Packages.tensorflow-datasets.dist</li>
    <li>python310Packages.treeo</li>
    <li>python310Packages.treeo.dist</li>
    <li>python310Packages.treex</li>
    <li>python310Packages.treex.dist</li>
    <li>python311Packages.distrax</li>
    <li>python311Packages.distrax.dist</li>
    <li>python311Packages.dm-sonnet</li>
    <li>python311Packages.dm-sonnet.dist</li>
    <li>python311Packages.elegy</li>
    <li>python311Packages.elegy.dist</li>
    <li>python311Packages.rlax</li>
    <li>python311Packages.rlax.dist</li>
    <li>python311Packages.tensorflow-datasets</li>
    <li>python311Packages.tensorflow-datasets.dist</li>
    <li>python311Packages.treeo</li>
    <li>python311Packages.treeo.dist</li>
    <li>python311Packages.treex</li>
    <li>python311Packages.treex.dist</li>
  </ul>
</details>
<details>
  <summary>166 packages built:</summary>
  <ul>
    <li>libretranslate (python311Packages.libretranslate)</li>
    <li>libretranslate.dist (python311Packages.libretranslate.dist)</li>
    <li>python310Packages.aeppl</li>
    <li>python310Packages.aeppl.dist</li>
    <li>python310Packages.aesara</li>
    <li>python310Packages.aesara.dist</li>
    <li>python310Packages.argos-translate-files</li>
    <li>python310Packages.argos-translate-files.dist</li>
    <li>python310Packages.argostranslate</li>
    <li>python310Packages.argostranslate.dist</li>
    <li>python310Packages.arviz</li>
    <li>python310Packages.arviz.dist</li>
    <li>python310Packages.augmax</li>
    <li>python310Packages.augmax.dist</li>
    <li>python310Packages.awkward</li>
    <li>python310Packages.awkward.dist</li>
    <li>python310Packages.bambi</li>
    <li>python310Packages.bambi.dist</li>
    <li>python310Packages.blackjax</li>
    <li>python310Packages.blackjax.dist</li>
    <li>python310Packages.chex</li>
    <li>python310Packages.chex.dist</li>
    <li>python310Packages.coffea</li>
    <li>python310Packages.coffea.dist</li>
    <li>python310Packages.correctionlib</li>
    <li>python310Packages.correctionlib.dist</li>
    <li>python310Packages.ctranslate2</li>
    <li>python310Packages.ctranslate2.dist</li>
    <li>python310Packages.dalle-mini</li>
    <li>python310Packages.dalle-mini.dist</li>
    <li>python310Packages.dask-awkward</li>
    <li>python310Packages.dask-awkward.dist</li>
    <li>python310Packages.dm-haiku</li>
    <li>python310Packages.dm-haiku.dist</li>
    <li>python310Packages.dm-haiku.testsout</li>
    <li>python310Packages.equinox</li>
    <li>python310Packages.equinox.dist</li>
    <li>python310Packages.faster-whisper</li>
    <li>python310Packages.faster-whisper.dist</li>
    <li>python310Packages.flax</li>
    <li>python310Packages.flax.dist</li>
    <li>python310Packages.gymnasium</li>
    <li>python310Packages.gymnasium.dist</li>
    <li>python310Packages.jax</li>
    <li>python310Packages.jax.dist</li>
    <li>python310Packages.jaxlib (python310Packages.jaxlib-build ,python310Packages.jaxlibWithoutCuda)</li>
    <li>python310Packages.jaxlib-bin</li>
    <li>python310Packages.jaxlib-bin.dist</li>
    <li>python310Packages.jaxlib.dist (python310Packages.jaxlib-build.dist ,python310Packages.jaxlibWithoutCuda.dist)</li>
    <li>python310Packages.jaxlibWithCuda</li>
    <li>python310Packages.jaxlibWithCuda.dist</li>
    <li>python310Packages.jaxopt</li>
    <li>python310Packages.jaxopt.dist</li>
    <li>python310Packages.jmp</li>
    <li>python310Packages.jmp.dist</li>
    <li>python310Packages.libretranslate</li>
    <li>python310Packages.libretranslate.dist</li>
    <li>python310Packages.mplhep</li>
    <li>python310Packages.mplhep.dist</li>
    <li>python310Packages.numpyro</li>
    <li>python310Packages.numpyro.dist</li>
    <li>python310Packages.objax</li>
    <li>python310Packages.objax.dist</li>
    <li>python310Packages.optax</li>
    <li>python310Packages.optax.dist</li>
    <li>python310Packages.optax.testsout</li>
    <li>python310Packages.pymc</li>
    <li>python310Packages.pymc.dist</li>
    <li>python310Packages.pytensor</li>
    <li>python310Packages.pytensor.dist</li>
    <li>python310Packages.skrl</li>
    <li>python310Packages.skrl.dist</li>
    <li>python310Packages.tensorflow-bin</li>
    <li>python310Packages.tensorflow-bin.dist</li>
    <li>python310Packages.translatehtml</li>
    <li>python310Packages.translatehtml.dist</li>
    <li>python310Packages.uproot</li>
    <li>python310Packages.uproot.dist</li>
    <li>python310Packages.vector</li>
    <li>python310Packages.vector.dist</li>
    <li>python310Packages.vqgan-jax</li>
    <li>python310Packages.vqgan-jax.dist</li>
    <li>python311Packages.aeppl</li>
    <li>python311Packages.aeppl.dist</li>
    <li>python311Packages.aesara</li>
    <li>python311Packages.aesara.dist</li>
    <li>python311Packages.argos-translate-files</li>
    <li>python311Packages.argos-translate-files.dist</li>
    <li>python311Packages.argostranslate</li>
    <li>python311Packages.argostranslate.dist</li>
    <li>python311Packages.arviz</li>
    <li>python311Packages.arviz.dist</li>
    <li>python311Packages.augmax</li>
    <li>python311Packages.augmax.dist</li>
    <li>python311Packages.awkward</li>
    <li>python311Packages.awkward.dist</li>
    <li>python311Packages.bambi</li>
    <li>python311Packages.bambi.dist</li>
    <li>python311Packages.blackjax</li>
    <li>python311Packages.blackjax.dist</li>
    <li>python311Packages.chex</li>
    <li>python311Packages.chex.dist</li>
    <li>python311Packages.coffea</li>
    <li>python311Packages.coffea.dist</li>
    <li>python311Packages.correctionlib</li>
    <li>python311Packages.correctionlib.dist</li>
    <li>python311Packages.ctranslate2</li>
    <li>python311Packages.ctranslate2.dist</li>
    <li>python311Packages.dalle-mini</li>
    <li>python311Packages.dalle-mini.dist</li>
    <li>python311Packages.dask-awkward</li>
    <li>python311Packages.dask-awkward.dist</li>
    <li>python311Packages.dm-haiku</li>
    <li>python311Packages.dm-haiku.dist</li>
    <li>python311Packages.dm-haiku.testsout</li>
    <li>python311Packages.equinox</li>
    <li>python311Packages.equinox.dist</li>
    <li>python311Packages.faster-whisper</li>
    <li>python311Packages.faster-whisper.dist</li>
    <li>python311Packages.flax</li>
    <li>python311Packages.flax.dist</li>
    <li>python311Packages.gymnasium</li>
    <li>python311Packages.gymnasium.dist</li>
    <li>python311Packages.jax</li>
    <li>python311Packages.jax.dist</li>
    <li>python311Packages.jaxlib (python311Packages.jaxlib-build ,python311Packages.jaxlibWithoutCuda)</li>
    <li>python311Packages.jaxlib-bin</li>
    <li>python311Packages.jaxlib-bin.dist</li>
    <li>python311Packages.jaxlib.dist (python311Packages.jaxlib-build.dist ,python311Packages.jaxlibWithoutCuda.dist)</li>
    <li>python311Packages.jaxlibWithCuda</li>
    <li>python311Packages.jaxlibWithCuda.dist</li>
    <li>python311Packages.jaxopt</li>
    <li>python311Packages.jaxopt.dist</li>
    <li>python311Packages.jmp</li>
    <li>python311Packages.jmp.dist</li>
    <li>python311Packages.mplhep</li>
    <li>python311Packages.mplhep.dist</li>
    <li>python311Packages.numpyro</li>
    <li>python311Packages.numpyro.dist</li>
    <li>python311Packages.objax</li>
    <li>python311Packages.objax.dist</li>
    <li>python311Packages.optax</li>
    <li>python311Packages.optax.dist</li>
    <li>python311Packages.optax.testsout</li>
    <li>python311Packages.pymc</li>
    <li>python311Packages.pymc.dist</li>
    <li>python311Packages.pytensor</li>
    <li>python311Packages.pytensor.dist</li>
    <li>python311Packages.skrl</li>
    <li>python311Packages.skrl.dist</li>
    <li>python311Packages.tensorflow-bin</li>
    <li>python311Packages.tensorflow-bin.dist</li>
    <li>python311Packages.translatehtml</li>
    <li>python311Packages.translatehtml.dist</li>
    <li>python311Packages.uproot</li>
    <li>python311Packages.uproot.dist</li>
    <li>python311Packages.vector</li>
    <li>python311Packages.vector.dist</li>
    <li>python311Packages.vqgan-jax</li>
    <li>python311Packages.vqgan-jax.dist</li>
    <li>tts</li>
    <li>tts.dist</li>
    <li>whisper-ctranslate2</li>
    <li>whisper-ctranslate2.dist</li>
    <li>wyoming-faster-whisper</li>
    <li>wyoming-faster-whisper.dist</li>
  </ul>
</details>

**Comment by GaetanLepage** [comment]
Well, I think that this update is now working.
@samuela, do you have any comment/request or is it actually good to merge ?

File: pkgs/development/python-modules/jaxlib/default.nix [hunk 1]
```diff
@@ -95,7 +95,7 @@ let
     "absl_py"
     "astor_archive"
     "astunparse_archive"
-    "boringssl"
+    # "boringssl"
```
Comment (samuela) [inline pkgs/development/python-modules/jaxlib/default.nix:98]:
```suggestion
```
let's just remove this line

**Review by samuela — CHANGES_REQUESTED** [review]
one small change and then i think we're good to go!

**Comment by BillHuang2001** [comment]
> > Delete this following patch seems to solve the problem, `jaxlib` will build, `jax` will also build, but failed in one unit test `tests/qdwh_test.py:testQdwhWithTinyElement`, but I don't think that's our problem.
> 
> This patch was indeed what was preventing the build to work.
> 
> I removed it and it seems to work fine. Actually, I haven't observed a single test failure.

Great, please go ahead and merge it.
About the test failure, I tried an AMD machine, and the issue is gone. But I can always reproduce the issue on the first machine (intel i5-13400F) and it's a numerical precision issue. I used `nix copy`, so the `jaxlib` binary files on two machines are the same. I guess that's probably MKL's problem, but we should open another issue for this, or report to upstream.

<details>

<summary>The output of the failed test</summary>

```bash
__________________ QdwhTest.testQdwhWithOnRankDeficientInput5 __________________
[gw3] linux -- Python 3.11.5 /nix/store/ffll6glz3gwx342z0ch8wx30p5cnqz1z-python3-3.11.5/bin/python3.11

self = <qdwh_test.QdwhTest testMethod=testQdwhWithOnRankDeficientInput5>, m = 10
n = 10, log_cond = 3.0

    @jtu.sample_product(
      [dict(m=m, n=n) for m, n in [(10, 10), (8, 8)]],
      log_cond=np.linspace(1, 4, 4),
    )
    def testQdwhWithOnRankDeficientInput(self, m, n, log_cond):
      """Tests qdwh with rank-deficient input."""
      a = np.triu(np.ones((m, n))).astype(_QDWH_TEST_DTYPE)
    
      # Generates a rank-deficient input.
      u, s, v = np.linalg.svd(a, full_matrices=False)
      cond = 10**log_cond
      s = jnp.linspace(cond, 1, min(m, n))
      s = jnp.expand_dims(s.at[-1].set(0), range(u.ndim - 1))
      a = (u * s) @ v
    
      is_hermitian = _check_symmetry(a)
      max_iterations = 15
      actual_u, actual_h, _, _ = qdwh.qdwh(a, is_hermitian=is_hermitian,
                                           max_iterations=max_iterations)
      _, expected_h = osp_linalg.polar(a)
    
      # For rank-deficient matrix, `u` is not unique.
      with self.subTest('Test h.'):
        relative_diff_h = _compute_relative_diff(actual_h, expected_h)
        np.testing.assert_almost_equal(relative_diff_h, 1E-6, decimal=5)
    
      with self.subTest('Test u.dot(h).'):
        a_round_trip = _dot(actual_u, actual_h)
        relative_diff_a = _compute_relative_diff(a_round_trip, a)
        np.testing.assert_almost_equal(relative_diff_a, 1E-6, decimal=5)
    
      with self.subTest('Test orthogonality.'):
        actual_results = _dot(actual_u.T.conj(), actual_u)
        expected_results = np.eye(n)
>       self.assertAllClose(
            actual_results, expected_results, rtol=_QDWH_TEST_EPS, atol=1e-6
        )

tests/qdwh_test.py:202: 
_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ 
jax/_src/test_util.py:1013: in assertAllClose
    self.assertArraysAllClose(x, y, check_dtypes=False, atol=atol, rtol=rtol,
jax/_src/test_util.py:978: in assertArraysAllClose
    _assert_numpy_allclose(x, y, atol=atol, rtol=rtol, err_msg=err_msg)
jax/_src/public_test_util.py:121: in _assert_numpy_allclose
    np.testing.assert_allclose(a, b, **kw, err_msg=err_msg)
_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ 

args = (<function assert_allclose.<locals>.compare at 0x7ffe0d589580>, array([[ 0.983452  ,  0.03162566, -0.04389316,  0.0522...., 0., 1., 0., 0.],
       [0., 0., 0., 0., 0., 0., 0., 0., 1., 0.],
       [0., 0., 0., 0., 0., 0., 0., 0., 0., 1.]]))
kwds = {'equal_nan': True, 'err_msg': '', 'header': 'Not equal to tolerance rtol=1.19209e-07, atol=1e-06', 'verbose': True}

    @wraps(func)
    def inner(*args, **kwds):
        with self._recreate_cm():
>           return func(*args, **kwds)
E           AssertionError: 
E           Not equal to tolerance rtol=1.19209e-07, atol=1e-06
E           
E           Mismatched elements: 100 / 100 (100%)
E           Max absolute difference: 0.18940455
E           Max relative difference: 0.18940455
E            x: array([[ 0.983452,  0.031626, -0.043893,  0.052261, -0.055985,  0.054734,
E                   -0.04862 ,  0.038186, -0.024359,  0.008367],
E                  [ 0.031626,  0.939559,  0.083886, -0.099878,  0.106995, -0.104604,...
E            y: array([[1., 0., 0., 0., 0., 0., 0., 0., 0., 0.],
E                  [0., 1., 0., 0., 0., 0., 0., 0., 0., 0.],
E                  [0., 0., 1., 0., 0., 0., 0., 0., 0., 0.],...

/nix/store/ffll6glz3gwx342z0ch8wx30p5cnqz1z-python3-3.11.5/lib/python3.11/contextlib.py:81: AssertionError
=========================== short test summary info ============================
FAILED tests/qdwh_test.py::QdwhTest::testQdwhWithOnRankDeficientInput5 - AssertionError:
```

</details>

File: pkgs/development/python-modules/jaxlib/default.nix (hunk 1 above)
Comment (GaetanLepage) [reply pkgs/development/python-modules/jaxlib/default.nix:98]: Done !

**Review by SomeoneSerge — APPROVED** [review]
@samuela I think github is still waiting for you to reset the "changes requested" status

**Comment by samuela** [comment]
The diff LGTM but I was waiting on 

> report to upstream

for the test failure. A decent number of folks use Intel CPUs so IMHO we might as well get ahead of this build failure. I propose that we report upstream and then add the test to `disabledTests` with a code comment linking to the upstream issue.

**Comment by BillHuang2001** [comment]
> for the test failure. A decent number of folks use Intel CPUs so IMHO we might as well get ahead of this build failure. I propose that we report upstream and then add the test to `disabledTests` with a code comment linking to the upstream issue.

Reported. https://github.com/google/jax/issues/18535

**Comment by GaetanLepage** [comment]
Thank you @BillHuang2001 
I added this test to the disabled ones.

**Comment by GaetanLepage** [comment]
Result of `nixpkgs-review pr 265417` run on x86_64-linux [1](https://github.com/Mic92/nixpkgs-review)
<details>
  <summary>28 packages marked as broken and skipped:</summary>
  <ul>
    <li>python310Packages.distrax</li>
    <li>python310Packages.distrax.dist</li>
    <li>python310Packages.dm-sonnet</li>
    <li>python310Packages.dm-sonnet.dist</li>
    <li>python310Packages.elegy</li>
    <li>python310Packages.elegy.dist</li>
    <li>python310Packages.rlax</li>
    <li>python310Packages.rlax.dist</li>
    <li>python310Packages.tensorflow-datasets</li>
    <li>python310Packages.tensorflow-datasets.dist</li>
    <li>python310Packages.treeo</li>
    <li>python310Packages.treeo.dist</li>
    <li>python310Packages.treex</li>
    <li>python310Packages.treex.dist</li>
    <li>python311Packages.distrax</li>
    <li>python311Packages.distrax.dist</li>
    <li>python311Packages.dm-sonnet</li>
    <li>python311Packages.dm-sonnet.dist</li>
    <li>python311Packages.elegy</li>
    <li>python311Packages.elegy.dist</li>
    <li>python311Packages.rlax</li>
    <li>python311Packages.rlax.dist</li>
    <li>python311Packages.tensorflow-datasets</li>
    <li>python311Packages.tensorflow-datasets.dist</li>
    <li>python311Packages.treeo</li>
    <li>python311Packages.treeo.dist</li>
    <li>python311Packages.treex</li>
    <li>python311Packages.treex.dist</li>
  </ul>
</details>
<details>
  <summary>166 packages built:</summary>
  <ul>
    <li>libretranslate (python311Packages.libretranslate)</li>
    <li>libretranslate.dist (python311Packages.libretranslate.dist)</li>
    <li>python310Packages.aeppl</li>
    <li>python310Packages.aeppl.dist</li>
    <li>python310Packages.aesara</li>
    <li>python310Packages.aesara.dist</li>
    <li>python310Packages.argos-translate-files</li>
    <li>python310Packages.argos-translate-files.dist</li>
    <li>python310Packages.argostranslate</li>
    <li>python310Packages.argostranslate.dist</li>
    <li>python310Packages.arviz</li>
    <li>python310Packages.arviz.dist</li>
    <li>python310Packages.augmax</li>
    <li>python310Packages.augmax.dist</li>
    <li>python310Packages.awkward</li>
    <li>python310Packages.awkward.dist</li>
    <li>python310Packages.bambi</li>
    <li>python310Packages.bambi.dist</li>
    <li>python310Packages.blackjax</li>
    <li>python310Packages.blackjax.dist</li>
    <li>python310Packages.chex</li>
    <li>python310Packages.chex.dist</li>
    <li>python310Packages.coffea</li>
    <li>python310Packages.coffea.dist</li>
    <li>python310Packages.correctionlib</li>
    <li>python310Packages.correctionlib.dist</li>
    <li>python310Packages.ctranslate2</li>
    <li>python310Packages.ctranslate2.dist</li>
    <li>python310Packages.dalle-mini</li>
    <li>python310Packages.dalle-mini.dist</li>
    <li>python310Packages.dask-awkward</li>
    <li>python310Packages.dask-awkward.dist</li>
    <li>python310Packages.dm-haiku</li>
    <li>python310Packages.dm-haiku.dist</li>
    <li>python310Packages.dm-haiku.testsout</li>
    <li>python310Packages.equinox</li>
    <li>python310Packages.equinox.dist</li>
    <li>python310Packages.faster-whisper</li>
    <li>python310Packages.faster-whisper.dist</li>
    <li>python310Packages.flax</li>
    <li>python310Packages.flax.dist</li>
    <li>python310Packages.gymnasium</li>
    <li>python310Packages.gymnasium.dist</li>
    <li>python310Packages.jax</li>
    <li>python310Packages.jax.dist</li>
    <li>python310Packages.jaxlib (python310Packages.jaxlib-build ,python310Packages.jaxlibWithoutCuda)</li>
    <li>python310Packages.jaxlib-bin</li>
    <li>python310Packages.jaxlib-bin.dist</li>
    <li>python310Packages.jaxlib.dist (python310Packages.jaxlib-build.dist ,python310Packages.jaxlibWithoutCuda.dist)</li>
    <li>python310Packages.jaxlibWithCuda</li>
    <li>python310Packages.jaxlibWithCuda.dist</li>
    <li>python310Packages.jaxopt</li>
    <li>python310Packages.jaxopt.dist</li>
    <li>python310Packages.jmp</li>
    <li>python310Packages.jmp.dist</li>
    <li>python310Packages.libretranslate</li>
    <li>python310Packages.libretranslate.dist</li>
    <li>python310Packages.mplhep</li>
    <li>python310Packages.mplhep.dist</li>
    <li>python310Packages.numpyro</li>
    <li>python310Packages.numpyro.dist</li>
    <li>python310Packages.objax</li>
    <li>python310Packages.objax.dist</li>
    <li>python310Packages.optax</li>
    <li>python310Packages.optax.dist</li>
    <li>python310Packages.optax.testsout</li>
    <li>python310Packages.pymc</li>
    <li>python310Packages.pymc.dist</li>
    <li>python310Packages.pytensor</li>
    <li>python310Packages.pytensor.dist</li>
    <li>python310Packages.skrl</li>
    <li>python310Packages.skrl.dist</li>
    <li>python310Packages.tensorflow-bin</li>
    <li>python310Packages.tensorflow-bin.dist</li>
    <li>python310Packages.translatehtml</li>
    <li>python310Packages.translatehtml.dist</li>
    <li>python310Packages.uproot</li>
    <li>python310Packages.uproot.dist</li>
    <li>python310Packages.vector</li>
    <li>python310Packages.vector.dist</li>
    <li>python310Packages.vqgan-jax</li>
    <li>python310Packages.vqgan-jax.dist</li>
    <li>python311Packages.aeppl</li>
    <li>python311Packages.aeppl.dist</li>
    <li>python311Packages.aesara</li>
    <li>python311Packages.aesara.dist</li>
    <li>python311Packages.argos-translate-files</li>
    <li>python311Packages.argos-translate-files.dist</li>
    <li>python311Packages.argostranslate</li>
    <li>python311Packages.argostranslate.dist</li>
    <li>python311Packages.arviz</li>
    <li>python311Packages.arviz.dist</li>
    <li>python311Packages.augmax</li>
    <li>python311Packages.augmax.dist</li>
    <li>python311Packages.awkward</li>
    <li>python311Packages.awkward.dist</li>
    <li>python311Packages.bambi</li>
    <li>python311Packages.bambi.dist</li>
    <li>python311Packages.blackjax</li>
    <li>python311Packages.blackjax.dist</li>
    <li>python311Packages.chex</li>
    <li>python311Packages.chex.dist</li>
    <li>python311Packages.coffea</li>
    <li>python311Packages.coffea.dist</li>
    <li>python311Packages.correctionlib</li>
    <li>python311Packages.correctionlib.dist</li>
    <li>python311Packages.ctranslate2</li>
    <li>python311Packages.ctranslate2.dist</li>
    <li>python311Packages.dalle-mini</li>
    <li>python311Packages.dalle-mini.dist</li>
    <li>python311Packages.dask-awkward</li>
    <li>python311Packages.dask-awkward.dist</li>
    <li>python311Packages.dm-haiku</li>
    <li>python311Packages.dm-haiku.dist</li>
    <li>python311Packages.dm-haiku.testsout</li>
    <li>python311Packages.equinox</li>
    <li>python311Packages.equinox.dist</li>
    <li>python311Packages.faster-whisper</li>
    <li>python311Packages.faster-whisper.dist</li>
    <li>python311Packages.flax</li>
    <li>python311Packages.flax.dist</li>
    <li>python311Packages.gymnasium</li>
    <li>python311Packages.gymnasium.dist</li>
    <li>python311Packages.jax</li>
    <li>python311Packages.jax.dist</li>
    <li>python311Packages.jaxlib (python311Packages.jaxlib-build ,python311Packages.jaxlibWithoutCuda)</li>
    <li>python311Packages.jaxlib-bin</li>
    <li>python311Packages.jaxlib-bin.dist</li>
    <li>python311Packages.jaxlib.dist (python311Packages.jaxlib-build.dist ,python311Packages.jaxlibWithoutCuda.dist)</li>
    <li>python311Packages.jaxlibWithCuda</li>
    <li>python311Packages.jaxlibWithCuda.dist</li>
    <li>python311Packages.jaxopt</li>
    <li>python311Packages.jaxopt.dist</li>
    <li>python311Packages.jmp</li>
    <li>python311Packages.jmp.dist</li>
    <li>python311Packages.mplhep</li>
    <li>python311Packages.mplhep.dist</li>
    <li>python311Packages.numpyro</li>
    <li>python311Packages.numpyro.dist</li>
    <li>python311Packages.objax</li>
    <li>python311Packages.objax.dist</li>
    <li>python311Packages.optax</li>
    <li>python311Packages.optax.dist</li>
    <li>python311Packages.optax.testsout</li>
    <li>python311Packages.pymc</li>
    <li>python311Packages.pymc.dist</li>
    <li>python311Packages.pytensor</li>
    <li>python311Packages.pytensor.dist</li>
    <li>python311Packages.skrl</li>
    <li>python311Packages.skrl.dist</li>
    <li>python311Packages.tensorflow-bin</li>
    <li>python311Packages.tensorflow-bin.dist</li>
    <li>python311Packages.translatehtml</li>
    <li>python311Packages.translatehtml.dist</li>
    <li>python311Packages.uproot</li>
    <li>python311Packages.uproot.dist</li>
    <li>python311Packages.vector</li>
    <li>python311Packages.vector.dist</li>
    <li>python311Packages.vqgan-jax</li>
    <li>python311Packages.vqgan-jax.dist</li>
    <li>tts</li>
    <li>tts.dist</li>
    <li>whisper-ctranslate2</li>
    <li>whisper-ctranslate2.dist</li>
    <li>wyoming-faster-whisper</li>
    <li>wyoming-faster-whisper.dist</li>
  </ul>
</details>

**Review by SomeoneSerge — APPROVED** [review]

**Review by samuela — APPROVED** [review]
woohoo! thanks @GaetanLepage ! 🎉

**Pull request merged**